README.md

```diff
@@ -15,12 +15,20 @@ To ensure the long-term health of the project, we are temporarily pausing all of
 
 ✅ We will continue reviewing and merging community PRs.
 
+✅ No CI Changes at all - Linting, Testing, e2e testing are all happening on every single pull request.
+
+✅ Security updates, patches, etc. will still be merged to the project (to the `main` branch)
+
 ✅ Contributions will be available on the main branch.
 
 ❌ We will not provide support via GitHub Discussions, Slack, or issue comments.
 
 ❌ We will not publish any new releases (major, minor, or patch), including 0.19.x and 1.0.x.
 
+  - This means no binaries, helm charts, attestation, etc. for any versioned release. Main branch and binaries produced with `main` tag will keep being bumped.
```

Comment:
Did you mean "We will not tag code going forward. As a result, no artifacts (binaries, helm charts, manifests, attestations, etc.) will be available to consume in a stable manner. Please produce your own artifacts or contribute to this team." ?

Reply:
I am confused about your comment compared to previous statement.

Reply:
Contributing to this team is not an alternative to get releases :smile: